dune/xt/common/bisect.hh

```diff
@@ -22,27 +22,31 @@ namespace Common {
 
 /**
  * \brief Finds the largest number x between left and right, where condition(x) is true, but condition(y) is false for
- *        any y < x.
+ *        any y > x.
  *
  * \note Presumes that: if condition(x_1) == condition(x_2) == value,
  *                      then there does not exist a x_1 < y < x_2, s.t. condition(y) == !value;
  */
-double find_largest_by_bisection(const double& left, const double& right, std::function<bool(const double&)> condition)
+double find_largest_by_bisection(const double& left,
+                                 const double& right,
+                                 std::function<bool(const double&)> condition,
+                                 const double rel_error = 1e-16,
```

Comment:
Maybe get those defaults from the FloatCmp defaults for double? That would be more consistent.

Reply:
Done. Also had to change the test, as ``EXPECT_DOUBLE_EQ`` seems to use a smaller default tolerance.

Reply:
There are shortcuts for that: `DXTC_EXPECT_FLOAT_EQ` and so forth from https://github.com/dune-community/dune-xt-common/blob/master/dune/xt/common/test/float_cmp_generated.hxx (which should not be included directly, but rather https://github.com/dune-community/dune-xt-common/blob/master/dune/xt/common/test/float_cmp.hh)

Reply:
Okay, used these shortcuts now.